dcs/mission.py

```diff
@@ -1844,6 +1844,56 @@ def find_group_by_id(self, group_id: int) -> Optional[Group]:
                 return g
         return None
 
+    def find_unit(self, unit_name: str, coalition: Optional[Coalition] = None) -> Optional[Unit]:
+        """Search for a unit matching a specified unit name. If a coalition is indicated,
+        the search is limited to that coalition; otherwise, it extends across all coalitions.
+
+        Args:
+            unit_name: unit's name as defined in the mission file
+            coalition: coalition to search in
+
+        Returns:
+            Unit: the unit found, otherwise None
+        """
+
+        coalitions = [coalition] if coalition is not None else self.coalition.values()
+        for k in coalitions:
+            for c in k.countries.values():
+                groups: List[Sequence[Group]] = [c.ship_group, c.vehicle_group, c.plane_group,
```

Comment:
Wouldn't it be simpler to just concat the group lists? instead of creating a list of groups?
e.g.:
```
groups: List[Group] = c.ship_group + c.vehicle_group....
```

Then it would be 1 for loop less

Reply:
@rp-  - no issues, will do it.